TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift

```diff
@@ -5,7 +5,7 @@ class TradeItSelectBrokerViewController: TradeItViewController, UITableViewDeleg
     @IBOutlet weak var brokerTable: UITableView!
     var delegate: TradeItSelectBrokerViewControllerDelegate?
     var alertManager = TradeItAlertManager()
-    var linkedBrokerManager: TradeItLinkedBrokerManager = TradeItLauncher.linkedBrokerManager
+    var linkedBrokerManager: TradeItLinkedBrokerManager = TradeItSDK.linkedBrokerManager
```

Comment:
Yeah that's fine - I thought you preferred this way so was sticking with the pattern